collect_app/src/main/java/org/odk/collect/android/database/helpers/FormsDatabaseHelper.java

```diff
@@ -55,7 +55,7 @@
     private static final String DATABASE_NAME = "forms.db";
     public static final String FORMS_TABLE_NAME = "forms";
 
-    static final int DATABASE_VERSION = 8;
+    public static final int DATABASE_VERSION = 8;
```

Comment:
There are a few places where I had to make constants/methods public that were previously package protected. If they weren't meant to be public then they really shouldn't have been used in the test but I didn't want to get into rewriting everything here.